packages/create-sitecore-jss/src/templates/angular/src/app/routing/routing.module.ts

```diff
@@ -39,32 +39,25 @@ const routes: Routes = [
     matcher: jssRouteMatcher,
     component: LayoutComponent,
     resolve: {
-      jssState: jssRouteResolver
+      jssState: jssRouteResolver,
     },
     runGuardsAndResolvers: 'always',
-  }
+  },
 ];
 
 @NgModule({
   imports: [
-    RouterModule.forRoot(routes, { onSameUrlNavigation: 'reload', initialNavigation: 'enabledBlocking' }),
+    RouterModule.forRoot(routes, {
+      onSameUrlNavigation: 'reload',
+      initialNavigation: 'enabledBlocking',
+    }),
     JssModule,
     TranslateModule,
-    BrowserModule
-  ],
-  exports: [
-    RouterModule,
-    TranslateModule,
-  ],
-  declarations: [
-    NotFoundComponent,
-    ServerErrorComponent,
-    LayoutComponent,
-    NavigationComponent,
-    VisitorIdentificationComponent
+    BrowserModule,
+    ScriptsModule,
   ],
-  providers: [
-    JssRouteBuilderService,
-  ]
+  exports: [RouterModule, TranslateModule],
+  declarations: [NotFoundComponent, ServerErrorComponent, LayoutComponent, NavigationComponent],
```

Comment:
Do we need to add `ScriptsModule` to declarations?

Reply:
Nope. In "declarations" we are only able to add "components", "directives", "pipes".
Since "ScriptsModule" is added to "imports", it means that all the components inside of "ScriptsModule" are available in all the templates inside the "RoutingModule"